src/Uno.UI.Tests/Windows_UI_Input/Given_GestureRecognizer.cs

```diff
@@ -827,13 +835,13 @@ protected void AreEquals(string name, ManipulationDelta? expected, ManipulationD
 				if (e.Translation.Y != actual.Translation.Y)
 					_pendingAssertResult.Add((name + ".Translation.Y", expected.Value.Translation.Y, actual.Translation.Y));
 
-				if (e.Rotation != actual.Rotation)
+				if (Math.Abs(e.Rotation - actual.Rotation) > .00001)
```

Comment:
Introduce a constant

Reply:
It's only for tests, it would only had some noise

Reply:
Oooh I thought it was production code.  Sorry.